content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Internal telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+You can monitor the health of any OpenTelemetry Collector instance by checking
+its own internal telemetry. Read on to learn how to configure this telemetry to
+help you [troubleshoot](/docs/collector/troubleshooting/) Collector issues.
+
+## Activate internal telemetry in the Collector
+
+By default, the Collector exposes its own telemetry in two ways:
+
+- Internal [metrics](#configure-internal-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configure-internal-logs) are emitted to `stdout` by default.
```

Comment:
I think we output logs to stderr by default https://github.com/open-telemetry/opentelemetry-collector/blob/d0f15e2463f8176e943c44fc7502f7ac80bacb0e/service/telemetry/config.go#L73

```suggestion
- [Logs](#configure-internal-logs) are emitted to `stderr` by default.
```